src/main/kotlin/net/atos/zac/documentcreation/DocumentCreationService.kt

```diff
@@ -124,6 +124,7 @@ class DocumentCreationService @Inject constructor(
                     zaak = zaak,
                     enkelvoudigInformatieObjectCreateLockRequest = it,
                     taskId = taskId,
+                    skipPolicyCheck = true
```

Comment:
I think it would be good to add a little code comment here explaining why we skip the policy check in this case? Other developers in future will want to know. :-)

Reply:
I agree. Will do

Reply:
Done